### ISSUE TITLE
Audit #1 – General Overview when logged in – Items not reachable or interactive by keyboard

### ISSUE THREAD

**Issue by marcus-herrmann** [owncloud/web#4333]
Connected to Audit 1, see #4300 
1.2 General Overview when logged in, URL: https://ocis-a11y.owncloud.works/#/files/list

# Issue
Files and folders (see screenshot) are not reachable via keyboard only. Mobile: Bread crumb dropdown not reachable via keyboard
![Xnip2020-11-18_13-05-08](https://user-images.githubusercontent.com/505181/99528515-b26d2100-299e-11eb-8d3e-78013b86938d.jpg)

Mobile (portrait mode) construct of collapsed bread crumbs is no button or link, just a div, even without a click event (a bug?)

"Rolle" Dropdown: Options are not keyboard accessible, only trigger is.

![CleanShot 2021-02-24 at 11 43 17](https://user-images.githubusercontent.com/505181/108989178-9bbd2100-7695-11eb-8aee-46aa09cf4d7c.png)

In Account menu, only the third item (a button) is reachable via keyboard:
![CleanShot 2021-02-24 at 12 33 14](https://user-images.githubusercontent.com/505181/108994861-9a432700-769c-11eb-9d58-894b108a1b92.png)

This is because Profil and Settings are incomplete links (just `<a><a/>` without `href` attribute). Some browsers don't put these incomplete/invalid links into the tabindex, making them unreachable.

# Remediation
Using (router-?)links for File and folders activator, which is currently just a clickable div. Using `a` or `button` should also remedy 2.5.2a.
Mobile (portrait mode) construct of collapsed bread crumbs: Depending on what is intended: disclosure widget? See: https://marcus.io/blog/disclosure-widget-vuejs
Using buttons for every single role option.
Using route links for Profil and Settings

**Comment by pascalwengerter** [comment]
Closing this since #5018 is merged and we're waiting for feedback (and will open new tickets with the requested changes from the `a11y` audit)
